README.md

```diff
@@ -389,6 +389,24 @@ spec:
       property: value
 ```
 
+Due to the way Azure handles binary files, you need to explicitly let the ExternalSecret know that the secret is binary.
+You can do that with the `isBinary` field on the key. This is necessary for certificates and other secret binary files.
+
+```yml
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: hello-keyvault-service
+spec:
+  backendType: azureKeyVault
+  keyVaultName: hello-world
+  data:
+    - key: hello-service/credentials
+      name: password
+      isBinary: "true"
```

Comment:
cant this just be `isBinary: true` and we add validation to it here https://github.com/godaddy/kubernetes-external-secrets/blob/master/crd.yaml#L62

Reply:
I have added it to the CRD with the type "boolean" and corrected the use of the field correspondingly.
I wasn't sure whether it should be added to the CRD since it is Azure specific, and I'm not that familiar with CRDs